### DISCUSSION TITLE
Deep links

### DISCUSSION THREAD

**Discussion by thisIsTheFoxe** [community/community#3075] [Mobile]
Hi, I just realized that if I tap a link to e.g. a specific comment in a conversation (e.g. an issue) I get redirected to the thread just fine but nothing indicates that a specific comment was meant to be in focus.. 
would be great to have that..
ty ^^

**Comment by aconsuegra** [comment]
Hey @thisIsTheFoxe,

thanks for the feedback! We are aware of some of the limitations we have with our deep links, specially when they include a comment or a line of code. We will be making some changes soon that hopefully will improve the experience for you.

Stay tuned!
